Add test for attributes (#77)
root-project/rntuple-validation: structure/attributes/README.md

# Attributes

## Fields

 * `Int32` (in main RNTuple)
 * `attr[std::int32_t]` (in attribute model)

> one regular and one attribute field.

Reviewer: ```suggestion
 * `Int32` (in main RNTuple)
 * `attr` (in attribute set called `Attributes`)

with each the type `std::int32_t`.
```

## Entries

> 1. Values for `attr`. One attribute entry refers to first main entry, a second one to next two main entries

Reviewer: ```suggestion
## Attribute Entries

1. Associated to the first entry of the main RNTuple
2. Referring to the second and third entry of the main RNTuple
```

# Attributes

## Fields

 * `Int32` (in main RNTuple)

>  * `attr[std::int32_t]` (in attribute set called `Attributes`)

Reviewer: ```suggestion
 * `attr` (in attribute set called `Attributes`)
```

# Attributes

## Fields

>  * `Int{32}`

Reviewer: ```suggestion
 * `Int32`
```

>  * `Attr`

Reviewer: In the code it's "attr" (lower-case). Maybe you can also make it explicit that `Int32` is the field in the main RNTuple, and `attr` is the field in the attribute model

> one regular and one attribute field.

Reviewer: This information is now in the list. However, we should describe the field types (`std::int32_t`)

## Entries

1. Attribute set name
2. Attribute set description

> 3. List of attribute entries with value and regular field range

Reviewer: These are not (RNTuple) entries. What you want to describe here is that there is one attribute entry spanning one main entry, and the second attribute entry spanning two main entries.

Reviewer: The first two items of the list are still there. This section of the README should describe what is written / read from the RNTuple data, ie "unroll" the third item.

# Attributes

> A RNTuple with a regular field and a attribute field.

Reviewer: Please describe at least the attribute name and its field in more detail, similar to the explanations for other tests.
